packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/QueryTests.kt

```diff
@@ -2213,6 +2213,34 @@ class QueryTests {
             }
     }
 
+    @Test
+    fun query_inListArgument() {
```

Comment:
I don't think we need to be exhaustive here, but it would be nice with a test for this that takes objects as arguments, just as a smoke test since it involves are few more moving parts.

Reply:
Actually we also need to test strings with `'` and `"`